server/user-groups.ts

```diff
@@ -92,7 +92,8 @@ export abstract class Auth extends Map<ID, GroupSymbol | ''> {
 		user: User,
 		permission: string,
 		target: User | GroupSymbol | null,
-		room?: Room | BasicRoom | null
+		room?: Room | BasicRoom | null,
```

Comment:
```suggestion
		room?: BasicRoom | null,
```

(`Room` is a subtype of `BasicRoom`, so you don't need both)